ccd-definition/AuthorisationCaseField/court-admin.json

```diff
@@ -123,21 +123,35 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "documents_socialWorkEvidenceTemplate_document",
     "UserRole": "caseworker-publiclaw-courtadmin",
-    "CRUD": "R"
+    "CRUD": "CRU"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "documents_threshold_document",
     "UserRole": "caseworker-publiclaw-courtadmin",
-    "CRUD": "R"
+    "CRUD": "CRU"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "documents_checklist_document",
     "UserRole": "caseworker-publiclaw-courtadmin",
-    "CRUD": "R"
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "courtBundle",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "displayCourtBundle",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "RU"
```

Comment:
is this definitely just RU?

Reply:
Going to the document page is mandatory for a valid case to be submitted so the field should be in the case data already by the time the admin can access it

Reply:
so why do they need U?

Reply:
It would change on the first time it is viewed by the admin from `No` to `Yes` so that update needs to be carried through

Reply:
No longer required in new implementation

Reply:
No longer required in current implementation